entc/gen/template/builder/query.tmpl

```diff
@@ -171,7 +171,7 @@ func ({{ $receiver }} *{{ $builder }}) OnlyX(ctx context.Context) *{{ $.Name }}
 }
 
 // OnlyID is like Only, but returns the only {{ $.Name }} ID in the query.
-// Returns a *NotSingularError when exactly one {{ $.Name }} ID is not found.
+// Returns a *NotSingularError when more than exactly one {{ $.Name }} ID is found.
```

Comment:
```suggestion
// Returns a *NotSingularError when more than one {{ $.Name }} ID is found.
```

Reply:
Thanks for the review @rotemtam. @iamnande, Can you please fix the PR and run codegen?